config/settings.yml

```diff
@@ -21,9 +21,9 @@ sdr:
     remote_output_path: /tmp
     remote_result_path: /tmp
     # Unix-style paths for the common-accessioning VM, used to manage files
-    local_ticket_path: /tmp
+    local_ticket_path: /tmp/input
     local_result_path: /tmp
-    local_output_path: /tmp
+    local_output_path: /tmp/output
     local_exception_path: /tmp
```

Comment:
differentiate between input and output folders so the tests for cleanup for be more specific